testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/opentracing/application/TracerConfigurationApplication.java

```diff
@@ -23,8 +27,37 @@
         @GET
         @Produces("text/plain")
         public String get() {
+            if (JaegerTracer.class.isInstance(this.tracer)) {
+                return tracer.toString() + getSenderConfiguration();
+            }
             return tracer.toString();
         }
 
+        private String getSenderConfiguration() {
+            try {
+                Field reporterField = this.tracer.getClass().getDeclaredField("reporter");
+                reporterField.setAccessible(true);
+                Object reporter = reporterField.get(this.tracer);
+                if ("RemoteReporter".equals(reporter.getClass().getSimpleName())) {
+                    Field senderField = reporter.getClass().getDeclaredField("sender");
+                    senderField.setAccessible(true);
+                    Object sender = senderField.get(reporter);
+                    if ("UdpSender".equals(sender.getClass().getSimpleName())) {
+                        Field transportField = sender.getClass().getDeclaredField("udpTransport");
+                        transportField.setAccessible(true);
+                        Object transport = transportField.get(sender);
+                        if ("ThriftUdpTransport".equals(transport.getClass().getSimpleName())) {
+                            Field socketField = transport.getClass().getDeclaredField("socket");
+                            socketField.setAccessible(true);
+                            DatagramSocket socket = (DatagramSocket) socketField.get(transport);
+                            return "sender-binding=" + NetworkUtils.formatIPAddressForURI(socket.getInetAddress()) + ":" + socket.getPort();
+                        }
+                    }
+                }
+            } catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException ex) {
+                ex.printStackTrace();
```

Comment:
Should we not be throwing the error here?

Reply:
yes, originally I wanted to avoid exception with different tracers implementations but this has been covered higher in the code